spaces/S000057/properties/P000162.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000057
+property: P000162
+value: true
+refs:
+- doi: 10.1007/978-1-4615-7819-2
+  name: Rings of Continuous Functions (Gillman and Jerison)
+---
+
+The identity function $\text{Id}:\mathbb{R}\to \mathbb{R}$, $\text{Id}(x) = x$ from {S000057} to {S000025} is a continuous bijection. Since every subspace of {S000025} is realcompact, {S000057} is realcompact from corollary 8.18 in {{doi:10.1007/978-1-4615-7819-2}}.
```

Comment:
I don't have access to `doi:10.1007/978-1-4615-7819-2` from my current location - I assume Cor 8.18 says if there is a continuous bijection from X to realcompact Y, then X is realcompact?

Reply:
Not quite, it says that every subspace of $Y$ is realcompact, and $f:X\to Y$ is a continuous injection, then $X$ is realcompact (and so is every subspace of $X$)

Reply:
Then I think the only thing I'd like to be reminded of is a reference for why every subspace (not just closed) of $\mathbb R^\kappa$ for suffiicently large $\kappa$ is realcompact.

Reply:
@StevenClontz That's not true of course. This only works for examples like $\mathbb{R}^n$ or $\mathbb{R}^\omega$ because they are separable metric spaces.

Reply:
Sorry, yes, I meant how large can $\kappa$ be while still guaranteeing every subspace is realcompact?

But by https://topology.pi-base.org/theorems/T000384 it's clear that any subspace of a separable metrizable space is realcompact, yes, so add a link there and I think this is ready.

Reply:
@StevenClontz It fails for $\kappa = \omega_1$ already, I had a question on math stack exchange because I was wondering this. Namely, $\omega_1$ embedds as a closed subspace of $\mathbb{R}^{\omega_1}$ but is not realcompact.

Reply:
See here https://math.stackexchange.com/questions/4730268/whats-kappa-min-nu-0-omega-1-text-embedds-into-mathbbr-nu